assets/src/dashboard/app/reducer/stories.js

```diff
@@ -17,7 +17,7 @@
 /**
  * Internal dependencies
  */
-import groupBy from '../../utils/groupBy';
+import { reshapeStoryObject } from '../api/useStoryApi';
```

Comment:
nit: now that `reshapeStoryObject` is happening in the reducer would it make more sense for this reshape to have it's own file. Something like `serializers/stories`?